iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java

```diff
@@ -98,15 +98,15 @@ public void apply(Network network, boolean throwException, ComputationManager co
             }
             return;
         }
-        if (switchPrefixId == null) {
-            switchPrefixId = voltageLevel1.getId();
-        }
         if (busOrBbs1 instanceof Bus && busOrBbs2 instanceof Bus) {
+            if (switchPrefixId == null) {
```

Comment:
Here the prefix Id can also simply be removed and the naming strategy should be used to define it

Reply:
that would mean removing the ability for the user to specify a name for the switch. Is it really what we want ?

Reply:
No you are right, I got lost here.